README.md

```diff
@@ -98,20 +98,24 @@ Generate Sphinx docs:
 The output will be written to `docs/_build/html/`.
 
 
-How to create a full-node in Ubuntu 16.04
+How to create a full-node in Ubuntu 20.04
 ------
 
 First, install all packages:
 
+    sudo add-apt-repository ppa:deadsnakes/ppa
     sudo apt update
-    sudo apt install --assume-yes python3 python3-dev python3-pip build-essential
-    sudo apt install --assume-yes supervisor  # optional
-    pip3 install -U pipenv
+    sudo apt install --assume-yes python3.6 python3.6-dev python3.6-pip build-essential virtualenv
+    sudo apt install --assume-yes supervisor  # optional    
 
 Then, install `hathor-python`:
 
     git clone git@gitlab.com:HathorNetwork/hathor-python.git
     cd hathor-python/
+    virtualenv -p /usr/bin/python3.6 venv
+    source venv/bin/activate
+    pip install -U pipenv grpcio_tools mypy-protobuf
+    pip install --upgrade protobuf 
```

Comment:
Essas duas linhas de cima não deveriam ser necessárias se em vez de `pipenv sync` for usado `pipenv sync -d`. Pode fazer essa mudança (tirar elas e melhorar o comando abaixo)?

Reply:
Então jansegre, o problema é que nao rolou, funcionaria se eu fizesse 

python3.6 -m pipenv sync 

tudo bem?

Reply:
O `pipenv sync -d` não rolou? Mas é o `pipenv` que não roda? O que falei é por conta do `-d` que é um atalho para `--dev` que é para incluir as dependências de desenvolvimento (que incluem o grpcio_tools, mypy-protobuf e protobuf que estão instalados manualmente no seu comando).